infra/pragma-node/Dockerfile

```diff
@@ -26,7 +26,7 @@ USER node:pragma
 WORKDIR /home/pragma-node
 
 COPY --from=builder --chown=node:pragma  /home/pragma-node/bin/pragma-node /usr/local/bin/pragma-node
-# COPY infra/pragma-node/config/.env.example .env
+COPY infra/pragma-node/config/.env .env
```

Comment:
you can let it commented, env values are filled from parameter store on AWS

Reply:
ah yesss, reverted